.github/workflows/build.yml

```diff
@@ -11,13 +11,7 @@ jobs:
         os:
           - macos-latest
           - ubuntu-latest
-          - windows-latest
         ocaml-version:
-          - 4.12.0
-          - 4.11.0
-          - 4.10.1
-          - 4.09.1
-          - 4.08.1
```

Comment:
github action is too slow if we're testing all these. Let's just focus on the only version that builds, and let's remove windows as support for windows is too hard anyway :P